e2etest/pkg/frr/config/config.go

```diff
@@ -18,30 +18,30 @@ import (
 // BGP router config.
 const bgpConfigTemplate = `
 router bgp {{.ASN}}
+  no bgp ebgp-requires-policy
```

Comment:
can u add the following tp prepare for Ipv6
```
no bgp default ipv4-unicast
```
I don't know why use next-hop-self it was in the original template and I replaced it with just active

Reply:
I think it'd be better to keep this PR focused on the current tests, but thanks for thinking ahead :smiley: